library/classes/Filtreatment_class.php

```diff
@@ -3,9 +3,9 @@
  * FILTREATMENT CLASS FILE
  *
  *
- * @author Cristian Năvălici {@link http://www.lemonsoftware.eu} lemonsoftware [at] gmail [.] com
+ * @author Cristian Năvălici {@link https://www.lemonsoftware.eu} lemonsoftware [at] gmail [.] com
```

Comment:
This link only works with http , so needs to stay as http

Reply:
👍